File: app/src/main/java/nic/goi/aarogyasetu/models/network/TokenValidationResponse.java
package nic.goi.aarogyasetu.models.network;


import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class TokenValidationResponse {

    @SerializedName("auth_token")
    @Expose
    private String authToken;
    @SerializedName("refresh_token")
    @Expose
    private String refreshToken;

    public String getAuthToken() {
        return authToken;
    }

    public String getRefreshToken() {
        return refreshToken;
    }

}

File: app/src/main/java/nic/goi/aarogyasetu/models/network/TokenValidationResponse.kt
package nic.goi.aarogyasetu.models.network


import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class TokenValidationResponse {

    @SerializedName("auth_token")
    @Expose
    val authToken: String? = null
    @SerializedName("refresh_token")
    @Expose
    val refreshToken: String? = null

}